sonar-db/src/main/java/org/sonar/db/DefaultDatabase.java

```diff
@@ -168,6 +180,6 @@ private static void completeDefaultProperty(Properties props, String key, String
 
   @Override
   public String toString() {
-    return "Database[" + (properties != null ? properties.getProperty(SONAR_JDBC_URL) : "?") + "]";
+    return format("Database[%s]", properties != null ? properties.getProperty(SONAR_JDBC_URL) : "?");
```

Comment:
there is no benefit to do that change in the first place either

don't know about the rule